src/amp/components/Epic.tsx

```diff
@@ -313,7 +492,145 @@ export const Epic: React.FC<{ webURL: string }> = ({ webURL }) => {
+								/>
+								<button
+									className={closeButtonStyle}
+									on="tap:AMP.setState({epicState:{hideReminderWrapper: true, hideButtons: false}}),reminderForm.clear"
+								>
+									<svg
+										viewBox="0 0 30 30"
+										xmlns="http://www.w3.org/2000/svg"
+									>
+										<path
+											fillRule="evenodd"
+											clipRule="evenodd"
+											d="M15.325 17.025l7.225 6.625 1.075-1.075-6.6-7.25 6.6-7.25L22.55 7l-7.225 6.625-7.25-6.6L7 8.1l6.625 7.225L7 22.55l1.075 1.075 7.25-6.6z"
+										/>
+									</svg>
+								</button>
+							</div>
+							<form
+								id="reminderForm"
+								data-amp-bind-hidden="epicState.hideReminderForm"
+								method="post"
+								action-xhr={setReminderUrl}
+								target="_blank"
+								custom-validation-reporting="interact-and-submit"
+								encType="application/x-www-form-urlencoded"
+								on="submit-success:AMP.setState({epicState:{hideReminderCta: true, hideSuccessMessage: false, hideReminderForm: true, headerText: 'Thank you! Your reminder is set.'}});submit-error:AMP.setState({epicState:{hideFailureMessage:false}})"
+							>
+								<div className={inputLabelStyle}>
+									Email address
+								</div>
+								<div
+									visible-when-invalid="typeMismatch"
+									className={invalidInputLabelStyle}
+									validation-for="reminderEmailAddress"
+								>
+									<svg
+										className={invalidInputSvgStyle}
+										viewBox="0 0 30 30"
+										xmlns="http://www.w3.org/2000/svg"
+									>
+										<path
+											fillRule="evenodd"
+											clipRule="evenodd"
+											d="M14.41 5L4 22.057l.668.943h20.664l.668-.943L15.59 5h-1.18zm-.063 12.178h1.306l.621-6.917-.856-.728h-.835l-.857.728.62 6.917zM15 18.452c.7 0 1.274.573 1.274 1.274 0 .7-.573 1.274-1.274 1.274-.7 0-1.274-.573-1.274-1.274 0-.7.573-1.274 1.274-1.274z"
+										/>
+									</svg>
+									Please enter a valid email address
+								</div>
+								<div
+									visible-when-invalid="valueMissing"
+									className={invalidInputLabelStyle}
+									validation-for="reminderEmailAddress"
+								>
+									<svg
+										viewBox="0 0 30 30"
+										xmlns="http://www.w3.org/2000/svg"
+										className={invalidInputSvgStyle}
+									>
+										<path
+											fillRule="evenodd"
+											clipRule="evenodd"
+											d="M14.41 5L4 22.057l.668.943h20.664l.668-.943L15.59 5h-1.18zm-.063 12.178h1.306l.621-6.917-.856-.728h-.835l-.857.728.62 6.917zM15 18.452c.7 0 1.274.573 1.274 1.274 0 .7-.573 1.274-1.274 1.274-.7 0-1.274-.573-1.274-1.274 0-.7.573-1.274 1.274-1.274z"
+										/>
+									</svg>
+									Please enter your email address
+								</div>
+								<input
+									type="hidden"
+									name="isPreContribution"
+									value="true"
+								/>
+								<input
+									type="hidden"
+									name="reminderDate"
+									value={`${reminderDay} ${reminderMonth} ${reminderYear}`}
```

Comment:
dotcom-components will need to transform this value before forwarding to the reminders api

Reply:
Yeah, I've just been adding the endpoint and remembered that myself. I'm going to change this.